core/shared/src/main/scala/zio/Cause.scala

```diff
@@ -335,6 +338,15 @@ sealed trait Cause[+E] extends Product with Serializable { self =>
   final def squash(implicit ev: E <:< Throwable): Throwable =
     squashWith(ev)
 
+  /**
+   * Squashes a `Cause` down to a single `Throwable`, chosen to be the
+   * "most important" `Throwable`.
+   * In addition, appends a new element the to `Throwable`s "caused by" chain,
+   * with this `Cause` "pretty printed" (in stackless mode) as the message.
+   */
+  final def squashWithTrace(f: E => Throwable): Throwable =
```

Comment:
Could you add `final def squashTrace(implicit ev: E <:< Throwable): Throwable` as well, for consistency with `squash`?